### ISSUE TITLE
relativeCaption plugin error

### ISSUE THREAD

**Issue by jengalas** [sachinchoolur/lightGallery#1046]
Thank you so much for implementing the relativeCaption plugin, and so quickly! I am missing something, though. I'm getting the error `Uncaught TypeError: Cannot read property 'getBoundingClientRect' of undefined` (see for example: https://planetzhanna.com/galleries/acadia-2013.html#lg=1&slide=0 ) 

I think I've included all necessary files and added the plugin when initializing the dynamic gallery. Is there a setting I need to add or change?

**Comment by sachinchoolur** [comment]
Hey,
I thought I'll inform you once I complete the documentation. Also, need to make few more improvements.

You need to wrap the captions with `.lg-relative-caption-item` and pass `appendSubHtmlTo: '.lg-item',` via settings.

```html

<div id="gallery-captions-demo">
    <a
        href="img/img1.jpg"
        data-sub-html="<div class='lg-relative-caption-item'><h4>Fading Light</h4><p>Classic view from Rigwood Jetty on Coniston Water an old archive shot similar to an old post but a little later on.</p></div>"
    >
        <img src="img/thumb1.jpg" />
    </a>
<div>
```

Let me know if you face any issues.

**Comment by jengalas** [comment]
Sorry, I just got excited when I saw that you had created the plugin so quickly! What you suggested is working fine for me, thank you. 

I will have to see how it works with vertical images, which are always tricky when dealing with captions, but most of my images are horizontal.

**Comment by sachinchoolur** [comment]
Hey @jengalas,
The issues with relativeCaption plugin have been resolved. Available in version `2.2.0-beta.5`
